src/main/java/org/sagebionetworks/web/client/widget/entity/file/AddToDownloadListV2.java

```diff
@@ -0,0 +1,10 @@
+package org.sagebionetworks.web.client.widget.entity.file;
+
+import org.sagebionetworks.repo.model.table.Query;
+
+import com.google.gwt.user.client.ui.IsWidget;
+
```

Comment:
Can configure to add files based on a query, or all children of a folder/project